Asgard.Console/ConnectionOptions.cs

```diff
@@ -45,7 +112,24 @@ internal void Initialise()
             };
 
             connect.Clicked += () => {
-                SelectedPort = _availablePorts[lv.SelectedItem];
+                this.Connection = new Communications.ConnectionOptions();
+                switch (rb.SelectedItem)
+                {
+                    case 0:
+                        Connection.ConnectionType = Communications.ConnectionOptions.ConnectionTypes.SerialPort;
+                        Connection.SerialPort = new Communications.SerialPortTransportSettings { PortName = _availablePorts[lv.SelectedItem] };
+                        break;
+                    case 1:
+                        var h = host?.Text.ToString() ?? "localhost";
+                        if (!short.TryParse(port.Text.ToString(), out var p))
+                        {
+                            p = 5550;
```

Comment:
The default port number should really be a constant rather than a hard-coded magic number.

Reply:
Now updated.